### PR TITLE
feat: use node 20 (VO-484)

### PR DESCRIPTION
**Pull request by acezard**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -32,13 +32,8 @@ jobs:
     - name: "Quality checks"
       stage: "prebuild"
```
Comment (cballevre) [inline .travis.yml:33]: I think we can take advantage of this by removing the stages to speed up CI

Comment (acezard) [reply .travis.yml]: I followed cozy-store, what do you think https://github.com/cozy/cozy-store/blob/master/.travis.yml

File: .travis.yml [hunk 2]
```diff
@@ -1,12 +1,12 @@
 language: node_js
-node_js: 16
+node_js:
+  - '20.11.0'
```
Comment (cballevre) [inline .travis.yml:3]: Why did specify a minor version of node ? 🤔

File: .travis.yml [hunk 3]
```diff
@@ -24,24 +24,13 @@ env:
...
     # ENCRYPTED_IV
     - secure: "iz1NO2t2ZSEpc7CMz2CtMNXyFa4FTLRZu9wPlnx1IwXHTworqJ8ENSAv4kDHOFYWXubHb52c2c3dMMLLZtHxRpTtnHRvEDCHzLdc6RP5wnoOLogtfSzD140g94+iPEaXfg6hJ/Wv1nE+iBTtfDTpRMy8e/i+InU4af3FcwqMJW1oroslgd9ubZV+KUuv4nHZNOxTW67lLUQ2iRI3b7Ewjo/zkhMgCGLz4lnSO6KnspgKQ6VUA+UeV+Xx28Y5mDpH2ouhfajji6xUHptoB9AG2WZf6GfpOOMB5gxSnGNf6BiVal/TyG0unMiUy6dij+6XYnSpmG6EWna7nR35M4PD7pIwiOzcvwVwvZ4j5GnXIoIOA12Wd9vP8iPmji509slgsut00qFbgU4Uxii8ElGuDdChGzdVXuSYUBLtX7ldMcYKLMckyn/wBnWndQrDNv5jDROms+8YxNIFbm4kRM3ynt1xMFtm2v43Cj/VQ+ZwFV0cr3Y6y4QoThBrnuFXHIXJfe8HsuYZaGPpa9Jf1SDRNut7ClmvGm5+Yf+gkG1Csm82perZ1SQBtyBdtEPF71zQ1ktWYduQrG6jLb4On5zcGZ+CmPdjOopNatq6AatpFYpuZkVyhTbyn626fRIOCgbyDkgSS6yBweUB5eq8EW7Fdu8jqB0fR5+WKISe7m6dpTg="
-stages:
-  - prebuild
-  - build
 jobs:
   include:
-    - name: "Quality checks"
-      stage: "prebuild"
+    - name: "Lint"
```
Comment (cballevre) [inline .travis.yml:29]: Perhaps `Quality checks` was more relevant given that we're not just doing a lint here

File: .travis.yml (hunk 1 above)
Comment (cballevre) [reply .travis.yml]: Nice 👍

**Review by cballevre — APPROVED** [review]

**Pull request merged**